tools/cmpbyt.ml

```diff
@@ -31,7 +31,7 @@ let rec cmpbytes ic1 ic2 len ofs =
   end
 
 let skip_section name =
-  name = "DBUG"
+  name = "DBUG" || name = "CRCS"
```

Comment:
This looks like a mistake. If your aim is perfectly reproducible builds, then certainly we should replace `cmpbyt` with plain old `cmp`. Also, `stripdebug` shouldn't be used to do two unrelated things (stripping debug info and moving/copying files around).

It's a code smell (admittedly not introduced by this PR) that `cmpbyt` and `stripdebug` have to be kept in sync when only one of them is really needed.

Reply:
> Also, `stripdebug` shouldn't be used to do two unrelated things (stripping debug info and moving/copying files around).

That's what the comment in `tools/Makefile` precisely says it's for (although I quickly find https://github.com/ocaml/ocaml/pull/340#issuecomment-165484317 from the when it was added)